crates/l2/prover/zkvm/interface/Cargo.toml

```
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-serde = { version = "1.0.203", features = ["derive"] }
+serde = { workspace = true, features = ["derive"] }
```

`zkvm_interface` is not in the workspace, so it cannot use dependencies with the workspace syntax.
We should go back to the previous declaration.
```suggestion
serde = { version = "1.0.203", features = ["derive"] }
```

---

Done in [ca0d2e5](https://github.com/lambdaclass/ethrex/pull/1680/commits/ca0d2e5d59161eddba37a778ff539916526c5ba0)